xarray/core/dataset.py

```diff
@@ -4480,17 +4572,21 @@ def merge(
             Value to use for newly missing values. If a dict-like, maps
             variable names (including coordinates) to fill values.
         combine_attrs : {"drop", "identical", "no_conflicts", "drop_conflicts", \
-                        "override"}, default: "override"
-            String indicating how to combine attrs of the objects being merged:
+                        "override"} or callable, default: "drop"
+            A callable or a string indicating how to combine attrs of the objects being
+            merged:
 
             - "drop": empty attrs on returned Dataset.
             - "identical": all attrs must be the same on every object.
             - "no_conflicts": attrs from all objects are combined, any that have
-              the same name must also have the same value.
+            the same name must also have the same value.
             - "drop_conflicts": attrs from all objects are combined, any that have
-              the same name but different values are dropped.
+            the same name but different values are dropped.
             - "override": skip comparing and copy attrs from the first dataset to
-              the result.
+            the result.
+
+            If a callable, it must expect a sequence of ``attrs`` dicts and a context object
+            as its only parameters.
```

Comment:
Something here is causing the `/home/docs/checkouts/readthedocs.org/user_builds/xray/checkouts/6612/xarray/core/dataset.py:docstring of xarray.core.dataset.Dataset.merge:37: WARNING: Bullet list ends without a blank line; unexpected unindent.` error in the docs build, but I can't immediately see where it is; https://readthedocs.org/projects/xray/builds/16916153/